app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/stubs/controllers/BusinessVerificationStubController.scala

```diff
@@ -30,20 +30,33 @@ class BusinessVerificationStubController extends InjectedController {
 
   def createBusinessVerificationJourney: Action[JsValue] = Action.async(parse.json) {
     implicit request =>
-      val continueUrl: String = (request.body \ "continueUrl").as[String]
-
-      Future.successful {
-        Created(Json.obj(
-          "redirectUri" -> (continueUrl + s"?journeyId=$businessVerificationJourneyId")
-        ))
+      val jsonBody = for {
+        _ <- (request.body \ "journeyType").validate[String]
+        origin <- (request.body \ "origin").validate[String]
+        _ <-  ((request.body \ "identifiers").head \ "saUtr").validate[String]
+        _ <- (request.body \ "entityType").validate[String]
+        continueUrl <- (request.body \ "continueUrl").validate[String]
+        _ <- (request.body \ "accessibilityStatementUrl").validate[String]
+      } yield (origin, continueUrl)
```

Comment:
As we are only interested in the origin and continueUrl properties you can remove the lines for journeyType, identifiers, entity type and accessibilityStatementUrl above. If we were building a representation of the BV submission data you would need these values, but we aren't.

Reply:
done